pkg/api/context.go

```diff
@@ -26,6 +27,9 @@ import (
 // Context carries values across API boundaries.
 type Context interface {
 	Value(key interface{}) interface{}
+	Deadline() (deadline time.Time, ok bool)
```

Comment:
I see-- in that case can you document in this interface exactly what each
member should be used for? Thanks.

On Fri, Oct 9, 2015 at 3:05 PM, Timothy St. Clair <notifications@github.com>
wrote:

> In pkg/api/context.go
> <https://github.com/kubernetes/kubernetes/pull/15371#discussion_r41682896>
> :
>
> > @@ -26,6 +27,9 @@ import (
> >  // Context carries values across API boundaries.
> >  type Context interface {
> >  	Value(key interface{}) interface{}
> > +	Deadline() (deadline time.Time, ok bool)
>
> Actually no, b/c in order for the api.Context to be passed through the
> storage layer they are needed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15371/files#r41682896>.
>